sdk/src/main/java/com/horizen/account/state/MessageProcessor.java

```diff
@@ -0,0 +1,17 @@
+package com.horizen.account.state;
+
+public interface MessageProcessor {
+    // Initialization is going to happen only once at genesis State creation.
+    // Common pattern: declare a new fake smart contract account in the View
+    void init(AccountStateView view) throws MessageProcessorInitializationException;
```

Comment:
It is not clear to me why we need the view as input parameter. For instance we will use this init() call to setup the fake smart contract address which is not dependent on the view

Reply:
In this example we should store the new Account in the view.
So the can be modified during the initialization.

Reply:
This is the place where e.g. the ForgerStakesProcessor can make sure that its account exists and is not declared as "empty" by setting some (arbitrary) code hash, right?

Reply:
right